<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SapWindow</class>
 <widget class="QMainWindow" name="SapWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SapWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbAuftragsNr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Auftragsnummer</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbMaterialNr">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MaterialNr</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbGesMenAuftrag">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gesamtmenge Auftrag</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbGesFailMenge">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ges. Fail Menge</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAuftragsNr">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>60091935</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbMaterialNr">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbGesAufMenge">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbGesFailMenge">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPing">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ping</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRead">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>READ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>SAPLib</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbGesAufMengeFlash">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbGesFailMengeFlash">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbMaterialNrFlash">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FlashDB-dev</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbAuftragsNr_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SAP Nr</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSapNr">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>604424</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbMaterialkurztext">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbMaterialkurztext">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Materialkurztext</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbMaterialkurztextFlash">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>290</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbReadMatKurzText">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>MaterialKurzText</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>260</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FlashDB-dev</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>SAPLib</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
